Unity/Pyx/Assets/Scripts/GazeGestureManager.cs

```diff
@@ -76,17 +72,31 @@ void Update()
         }
     }
 
-    void ShowNextJoke()
+    void FetchJoke()
     {
-        jokeCount++;
+        var webRequest = UnityWebRequest.Get("http://192.168.1.153:5002/api/instruction/" + jokeIndex);
```

Comment:
I couldn't find a way to set a host record for the HoloLens. That would be super useful here!